lambda_functions/nightly_stats_bigquery_upload/nightly_stats_bigquery_upload_lambda.py

```diff
@@ -76,26 +108,49 @@ def add_updated_rows_for_date(
 def lambda_handler(
     event,
     _context,
-):
+) -> dict[str, int]:
```

Comment:
Did we want to log the table_id somewhere in here so we know which table is being used.